pkgs/build-support/rust/hooks/default.nix

```diff
@@ -75,11 +75,8 @@ in {
           ${lib.optionalString (stdenv.buildPlatform.config != stdenv.hostPlatform.config) ''
             [target."${shortTarget}"]
             "linker" = "${ccForHost}"
-            ${# https://github.com/rust-lang/rust/issues/46651#issuecomment-433611633
-            lib.optionalString (stdenv.hostPlatform.isMusl && stdenv.hostPlatform.isAarch64) ''
-              "rustflags" = [ "-C", "target-feature=+crt-static", "-C", "link-arg=-lgcc" ]
-            ''}
```

Comment:
Do we no longer need the `-lgcc` on pkgsStatic aarch64?

Reply:
Maybe you could explain why in the commit message? It doesn't provide any background at the moment.

Reply:
Nope, that is resolved upstream now, maybe by [this commit](https://github.com/rust-lang/compiler-builtins/commit/9945ce884afd2938c125e61fb2f74c9798d35f40)?
I tested pkgsCross.aarch64-multiplatform.pkgsStatic.fd to confirm.

Reply:
Maybe because https://github.com/rust-lang/compiler-builtins/pull/377 was merged?

Reply:
Just having exactly what you wrote here in the commit message would be a big help to future history readers. :)

Reply:
You're totally right, I just added more elaborate commit messages to multiple commits